src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int INVALID_AGE = 16;
+    private static final int VALID_AGE = 18;
+    private static final long VALID_ID = 1234567890L;
+    private static final String INVALID_PASSWORD = null;
+    private static final String VALID_PASSWORD = "qwerty";
+    private static final String VALID_LOGIN = "nameEx";
+    private static final String INVALID_LOGIN = "name";
+
+    private final User user = new User();
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
* user should not be final, and should be initialized before each test to achieve a proper isolation
* proper order of the fields:
  * constants
  * services
  * test data

```suggestion
    private final RegistrationService registrationService = new RegistrationServiceImpl();
    
    private User user;
```